### PR TITLE
Update cypress tests for redesigned main page

### PR DESCRIPTION
**Pull request by tfettig22**
What's this PR do?
- Update cypress tests for movie_container.cy.js
- Added the intercept and swiper functionality tests
- All tests passing for movie_container.cy.js

- [x] I tested with Mocha/Chai
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new bugs
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing E2E tests pass locally with my changes

### CODE REVIEW

File: cypress/e2e/single_movie.cy.js [hunk 1]
```diff
@@ -26,18 +26,13 @@ describe('empty spec', () => {
     })
   })
 
-  it('should be able to visit the page and find the header', () => {
-    cy.get('h1')
-    .contains('Rancid Tomatillos')
+  it('should be able to visit the page and find the X button to return to the main page', () => {
+    cy.get('[class=back-to-home-btn')
+    .contains('X')
   })
```
Comment (Maia-Cochran) [inline cypress/e2e/single_movie.cy.js:32]: I see we decided to get rid of the header for the single  movie view... I like the way this looks!

File: src/components/MovieContainer/MovieContainer.js [hunk 2]
```diff
@@ -91,11 +92,12 @@ const MovieContainer = ( {movies} ) => {
         modules={[Parallax, Pagination, Navigation, Mousewheel, Keyboard]}
         className="mySwiper2"
         slidesPerView={10}
-        slidesPerGroup={6}
+        slidesPerGroup={5}
```
Comment (Maia-Cochran) [inline src/components/MovieContainer/MovieContainer.js:95]: I'd really love to go over this piece together so I understand what's going on with the carousel :)

**Review by Maia-Cochran — APPROVED** [review]
Great work, can't wait to review this together. Looks really great overall, and I'm so interested to see the inner workings of the carousel!

**Review by GOECHA — APPROVED** [review]
Cypress testing looks great. Great job with funcitonality!

**Pull request merged**